### DISCUSSION TITLE
How do I disable cursor blinking?

### DISCUSSION THREAD

**Discussion by mawkler** [ghostty-org/ghostty#3413] [Q&A] [answered]
I've tried setting each these options, but none of them disables cursor blink:

```
cursor-style-blink = false
cursor-style-blink = true
cursor-style-blink = ""
```

**Comment by unikitty37** [comment]
It would be awesome if there was a force-disable option for this, too — I don't want apps to be able to turn the blinking cursor _on_ (especially if they forget to turn it off again).

Blinking should be entirely a user choice, as it's Extremely Distracting for some folk.

**Comment by gpanders** [comment, marked as answer]
Duplicate of https://github.com/ghostty-org/ghostty/discussions/2812
